### PR TITLE
Improve build system

### PR DESCRIPTION
**Pull request by k105la**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
   "description": "A Javascript port of the PySyft library",
   "main": "lib/index.js",
+  "module": "lib/index.esm.js",
   "scripts": {
     "start": "rollup -cw",
     "build": "rollup -c",
     "precommit": "npm run build && pretty-quick --staged",
     "test": "jest"
   },
+  "presets": ["env"],
```
Comment (cereallarceny) [inline package.json]: Do we need this defined twice?  Reference: https://github.com/OpenMined/syft.js/blob/master/.babelrc

File: rollup.config.js [hunk 2]
```diff
@@ -2,12 +2,19 @@ import babel from 'rollup-plugin-babel';
 
 export default {
   input: 'src/index.js',
-  output: {
+  output: [{
     file: 'lib/index.js',
     name: 'Syft',
     format: 'iife',
```
Comment (cereallarceny) [inline rollup.config.js]: Is IIFE necessary anymore?

Comment (k105la) [reply rollup.config.js]: No its not should be UMD

**Review by cereallarceny — APPROVED** [review]
This all works fine.  UMD covers everything via a script tag... ESM allows for ES6 import/export functionality.

**Pull request merged**